alacritty/src/config/bindings.rs

```diff
@@ -327,6 +327,10 @@ pub enum ViAction {
     InlineSearchNext,
     /// Jump to the previous inline search match.
     InlineSearchPrevious,
+    /// Search forward for highlighted region or word under cursor.
+    SearchRegionForward,
+    /// Search backward for highlighted region or word under cursor.
+    SearchRegionBackward,
```

Comment:
I find the name "region" extremely confusing here, why did you name it that way? We already have semantic bindings, iirc they're just called "…SemanticWord…"

Reply:
There's no `SemanticWord` binding, but there are `SemanticDir` motions.  I agree region is a bit weird but the only time we search for the semantic word is if we're in normal mode.  In the case of visual mode we want to search for the highlighted selection, so i chose "region" as a sort of catch-all

Reply:
does `SearchCursorForward` and `SearchCursorBackward` make more sense?